### DISCUSSION TITLE
how to short or make custum url

### DISCUSSION THREAD

**Discussion by nikhilmzp** [datarhei/restreamer#696] [General]
rtmp://localhost/nik/84fcba4b-3e76-4899-9b78-02f28e2d14fe.stream?token=nik. can i make this
rtmp://localhost/nik/live/nikhil

**Comment by ioppermann** [comment]
This is currently not possible. We'll add it to our [Feature Requests](https://github.com/datarhei/restreamer/discussions/345).

**Comment by nikhilmzp** [comment]
hey is there any option to add additional ffmpeg command like i need to add
logo in particular stream so  i can send it using ffmpeg?
